tests/integration/multicluster/helper.go

```diff
@@ -47,6 +47,10 @@ func Setup(controlPlaneValues *string, clusterLocalNS, mcReachabilityNS *namespa
 
 		// Set the cluster-local namespaces in the mesh config.
 		*controlPlaneValues = fmt.Sprintf(`
+values:
+  global:
+    meshExpansion:
+      enabled: true
```

Comment:
I'm not sure if this is necessary, it should be enabled as part of the multicluster install https://github.com/istio/istio/blob/bb809f91096d9abf33a69b532d1c2cf4338099f6/pkg/test/framework/components/istio/operator.go#L436

Reply:
Yes, I am just experimenting around. This is far from ready. Thanks for pointing it out, though!